src/lexitools/commands/correspondences.py

```diff
@@ -71,18 +71,28 @@ def register(parser):
         type=str,
         help='select a concept list to filter on')
 
+
 def iter_phonemes(tokens):
     for segment in tokens:
         if "/" in segment:
             segment = segment.split("/")[1]
         yield segment
 
 
+def get_langgenera_mapping(path):
+    langgenera = {}
+    with open(path) as csvfile:
+        reader = csv.reader(csvfile, delimiter='\t')
+        header = next(reader)
+        for row in reader:
+            row_dict = dict(zip(header, row))
+            langgenera[row_dict["GLOTTOCODE"]] = row_dict["GENUS"]
+    return langgenera
```

Comment:
BTW: if you have CLDF /lexibank packages installed then you can use csvw:

```python
def get_langgenera_mapping(path):
    with csvw.UnicodeDictReader(path, delimiter="\t") as reader:
        return {row['GLOTTOCODE']: row['GENUS'] for row in reader}
```

Reply:
Thanks, this is more practical indeed